tests/integrations/predefined_document/test_audio.py

```diff
@@ -29,7 +29,6 @@
     str(TOYDATA_DIR / 'hello.ogg'),
     str(TOYDATA_DIR / 'hello.wma'),
     str(TOYDATA_DIR / 'hello.aac'),
-    str(TOYDATA_DIR / 'hello'),
```

Comment:
why removed?

Reply:
Because it's an Audio URL without an audio extension and should not be validated

Reply:
I am not sure about that, (at least on unix) i can store an audio file without extension I believe, why should that not be allowed? Admittedly, for audio it may not be common to do that, but text files do it, e.g. `Dockerfile`

Reply:
Yes you can, but in order to avoid issues like https://github.com/docarray/docarray/issues/1555 we need to look at extensions. 

But you have a good point, text files without extensions are very common. Does it make sense to ignore validating text URLs that have no extensions? I don't really have another solution, we can't guess extensions or types in that scenario, and trying to read them during validation will be slow

Reply:
let's ignore validation of TextURL then?

Reply:
Can't we have a rule that is like "_if_ there is an extension, validate it; if there is no extension, don't validate"? We could have that for all url types, no?

Reply:
> "if there is an extension, validate it; if there is no extension, pass validation"

yeap this is what I meant, but for text urls only. 
ok let's do it for all urls